CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Introduction
+
+## Thanks
+
+Thank you for considering contributing to **GrPPI. Our aim is to make **GrPPI** useful for the software comunity at large. 
```

Comment:
Replace `comunity` by `community`. 

Please, consider the use of `ispell` before committing MD files. This would help spotting most of the spelling mistakes.